modules/default-requirements-secured-cluster-services.adoc

```diff
@@ -166,25 +168,25 @@ The requirements in this table are based on the default of 2 replicas.
 | Scanner V4 Indexer | CPU | Memory
 | *Request*
 | 2 cores
-| 3000 MiB
+| 3 GiB
+
 | *Limit*
 | 4 cores
 | 6 GiB
 |===
 
-Scanner V4 requires Scanner V4 DB to store data. The following table lists the minimum memory and storage values required to install and run Scanner V4 DB. For Scanner V4 DB, a PVC is strongly recommended because it ensures optimal performance. The PVC should be 10 GiB.
-
-[cols="3",options="header"]
+Scanner V4 requires Scanner V4 DB to store data. The following table lists the minimum memory, CPU, and storage values required to install and run Scanner V4 DB. For Scanner V4 DB, a PVC is strongly recommended (but not required) because it ensures optimal performance.
```

Comment:
```suggestion
Scanner V4 requires Scanner V4 DB to store data. The following table lists the minimum memory, CPU, and storage values required to install and run Scanner V4 DB. For Scanner V4 DB, a PVC is not required, but is strongly recommended because it ensures optimal performance.
```
IBM Style recommends to avoid parenthesis, does this sound OK?

Reply:
updated but with slightly different wording from the suggestion. Let me know what you think!